lib/index.js

```diff
@@ -63,4 +105,130 @@ Playground.Piezo = {
   }
 };
 
+Playground.parseFirmataByte = function(data) {
+  // Parse a byte value from two 7-bit byte firmata response bytes.
+  if (data.length != 2) {
+    throw('Expected 2 bytes of firmata response for a byte value!');
+  }
+  return (data[0] & 0x7F) | ((data[1] & 0x01) << 7);
+};
```

Comment:
This is mostly duplicating `Board.decode(data)`, with the main difference that it produces just the byte, not an array of decoded bytes. 

What do you think about this: 

```js
return Board.decode(data).shift();
```